extensions/amp-analytics/0.1/vendors/gtag.js

```diff
@@ -38,7 +37,7 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
          'sr=${screenWidth}x${screenHeight}&' +
          'cid=${clientId}&' +
          'tid=${trackingId}&' +
-         'dl=${documentLocation}&' +
+         'dl=${sourceUrl}&' +
```

Comment:
hmm, it does not seem to change behavior.

or, is your backend rewriting `sourceUrl` instead of `documentLocation`?

Reply:
That's correct. Plus adwords uses sourceUrl, so I'd rather keep them consistent.